frontend/src/components/Collections/components/DeleteCollection/index.tsx

```diff
@@ -22,7 +22,7 @@ const DeleteCollection: FC<Props> = ({
   Button = RawButton,
   isRevision,
 }) => {
-  const [deleteMutation, { isLoading }] = useDeleteCollection(id);
+  const { mutateAsync: deleteMutation, isLoading } = useDeleteCollection(id);
```

Comment:
https://react-query.tanstack.com/guides/migrating-to-react-query-3#mutationmutate-no-longer-return-a-promise